src/lib/helpers.ts

```diff
@@ -99,3 +101,13 @@ export const buildStatusBarItem = (text: string) => {
   statusBarItem.text = text;
   return statusBarItem;
 };
+
+export function showMissingAccountError() {
```

Comment:
I like this idea, but I'm not sure what to do to get it to show. I removed my hubspot.config.yml file but I'm not seeing this error surfacing anywhere. Do you know a way for me to test it?